app/controllers/api/v1/organizations_controller.rb

```diff
@@ -3,6 +3,14 @@
 module Api
   module V1
     class OrganizationsController < ApplicationController
+      before_action :authorization
+
+      # GET /organization_id/public
```

Comment:
This comment may not be needed. In Ruby is better to avoid comments unless they are explicitly needed.